data/categories.txt

```diff
@@ -43,7 +43,7 @@ nl:Eten
 fi:Ruoka
 fr:Nourriture
 de:Essen
-hu:Élelmiszer
+hu:Étkezés
```

Comment:
Is this more used than what we had when searching for "Food"?

Reply:
Élelmiszer is food in the sense that it is a product which you buy and eat. If you tap on an icon "Élelmiszer" you expect shops where you can buy food (e.g. a supermarket or a smaller convenience store). However, in maps.me it refers to places where you can eat, i.e. restaurants, cafés and so on. So it is more suitable to have "Étkezés" (literally eating) there than "Élelmiszer" which is  bit misleading.